src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/component/RecallAuditComponentTest.kt

```diff
@@ -83,16 +86,31 @@ class RecallAuditComponentTest : ComponentTestBase() {
   }
 
   @Test
-  fun `get audit summary for a recall`() {
+  fun `get reasonsForRecall (Array value) audit history`() {
```

Comment:
Maybe in a later test but would be good to see coverage for edits not just inserts.